apps/storybook/src/stories/core.stories.tsx

```diff
@@ -89,21 +89,21 @@ export const WorkspaceList: StoryFn = () => {
   return <FakeApp />;
 };
 WorkspaceList.play = async ({ canvasElement }) => {
+  const canvas = within(canvasElement);
   // click current-workspace
-  await waitFor(
+  const currentWorkspace = await waitFor(
     () => {
-      assertExists(
-        canvasElement.querySelector('[data-testid="current-workspace"]')
-      );
+      assertExists(canvas.getByTestId('current-workspace'));
+      return canvas.getByTestId('current-workspace');
     },
     {
       timeout: 5000,
     }
   );
-  const currentWorkspace = canvasElement.querySelector(
-    '[data-testid="current-workspace"]'
-  ) as Element;
-  await userEvent.click(currentWorkspace);
+
+  // todo: figure out why userEvent cannot click this element?
+  // await userEvent.click(currentWorkspace);
+  currentWorkspace.click();
```

Comment:
Is this cause screen snapshot change ?

Reply:
I am not sure. If you try it locally, `userEvent.click` on currentWorkspace has no effect at all, which is strange.